frontend/src/shared-components/donation-info/donation.info.jsx

```diff
@@ -1,31 +1,57 @@
 import React from "react";
-
+import { PaymentSlip } from "components";
 import styles from "./donation.info.module.scss";
 
-export const DonationInfo = ({ title, categories, text, donationInfos }) => {
+export const DonationInfo = ({ donation }) => {
+  let financialInfo = donation.financial_info
```

Comment:
Ovde mozes iskoristiti elvis operator ?. i ne treba ti onda if else null. Medjutim da li nam ovo treba s obzirom da koristis map financialInfo (value, index)

Reply:
Nije mi najjasnije na sta mislis pod `Medjutim da li nam ovo treba s obzirom da koristis map financialInfo (value, index)`, jer ta sekcija uopste ne treba da se pojavi ako je robna donacija posto nema racuna za uplatu.

Reply:
Aha, nisam razumeo kod uopste. Ovde ti definitivno ne treba if else, jer ga validiras dole u kodu na liniji 28. Ostavi ovde samo onda const financialInfo = pa ova lista koju si ovako raspodelio.
Ako ti baca error, onda samo na liniji 32 mu kaze financialInfo?.map -> elvis na map i to je to

Reply:
Vazi, sad cu pogledati da ispravim te freske 😀